cloudcompose/cluster/cloudinit.py

```diff
@@ -10,11 +10,10 @@ def __init__(self, cloud_config, base_dir='.'):
         self.template_file = 'cluster.sh'
 
     def build(self, **kwargs):
-        config_data = self.cloud_config.config_data('cluster')
+        config_data = self.cloud_config.config_data('cluster')[0]
```

Comment:
So the cloud_config.config_data returns this tuple
`return config_data[plugin_name], config_dir`
i wasn't sure if you were using config_dir so i added that index to get the config data i needed.

this is yml file im testing with:
```
cluster:
  name: some-super-tester-instance
  search_path:
    - templates
  aws:
    ami: ami-61bbf104
    username: centos
    terminate_protection: false
    security_groups: 
      - sg-f4190393
    vpc: vpc-c01498a5
    ebs_optimized: false
    instance_type: t2.small
    keypair: loxodo
    volumes:
      - name: root
        size: 30G
      - name: docker
        size: 20G
        block: /dev/xvdz
        file_system: lvm2
        meta:
          group: docker
          volumes:
            - name: data 
              size: 19G
            - name: metadata
              size: 900M 
      - name: data
        size: 10G
        block: /dev/xvdc
        file_system: ext4
        meta:
          format: true
          mount: /data/test
    tags:
      datadog: monitored      
    asg:
      subnets: subnet-11111111, subnet-2222222
      redundancy: 2
```